addon/install.rdf

```diff
@@ -8,8 +8,8 @@
     <em:hasEmbeddedWebExtension>false</em:hasEmbeddedWebExtension>
     <em:unpack>false</em:unpack>
     <em:version>1.0.0</em:version>
-    <em:name>Shield Bootstrap Example</em:name>
-    <em:description>&lt;description&gt;
+    <em:name>Shield Study RAPPOR</em:name>
+    <em:description>RAPPOR as a Shield Study
```

Comment:
Be a little more descriptive here and in the name. This study is about collecting eTLD+1 so "RAPPOR Shield study to collect homepage's eTLD+1"